modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/manager/v1_0/DefaultObjectEntryManagerImpl.java

```diff
@@ -864,6 +864,170 @@ private Map<String, String> _addAction(
 			actionName, methodName, serviceBuilderObjectEntry, null, uriInfo);
 	}
 
+	private com.liferay.object.model.ObjectEntry _addNestedObjectEntries(
```

Comment:
this code repeats a lot of stuff from _addOrUpdateNestedObjectEntries, maybe there's a way to refactor so the repetitive chunk of code is extracted? It is only an idea as I'm not sure if it is the best practice 😅